### ISSUE TITLE
Unable to use with TextToSpeech

### ISSUE THREAD

**Issue by EnesKaraosman** [Cay-Zhang/SwiftSpeech#2]
Firstly, the library is awesome.

But I bumped an issue when trying to use this with the text to speech.

Simply you can also reproduce the issue with the following code;

```swift
import AVFoundation
func onSpeechToTextEnded() {
   let utterance = AVSpeechUtterance(string: "Hello world")
   utterance.voice = AVSpeechSynthesisVoice(language: "en-GB") 

   let synthesizer = AVSpeechSynthesizer()
   synthesizer.speak(utterance)
}
```

if I try to call this function (onSpeechToTextEnded) before actually using this library, I can hear the voice.
But when I try calling this function to hear some voices it is now working.

Can you investigate the issue please

**Comment by EnesKaraosman** [comment]
I created a Pull Request, seems it solves the problem, after you investigate as well. 

Please consider accepting changes, thanks a lot again. 👍🏻

**Comment by Cay-Zhang** [comment]
Hi, thanks for your pull request but I think I will solve the problem by allowing users to choose from different audio session categories. The `.record` audio session category used in the library now silences virtually all output on the system for as long as the session is active, and it could be what the users expect.

I will add an option to choose an audio session category in `SwiftSpeech.Session.Configuration` later.

Thanks again for submitting the issue and pull request!

**Comment by EnesKaraosman** [comment]
Yes better to do it like you suggest, I also realized now. Also sounds now is quite low. so it is related with other parameter as well.

**Comment by Cay-Zhang** [comment]
The issue is fixed in [v0.9.2](https://github.com/Cay-Zhang/SwiftSpeech/releases/tag/v0.9.2). You can use `SwiftSpeech.Session.AudioSessionConfiguration.playAndRecord` to achieve the behavior you want 😉

Also, [SwiftSpeech Examples](https://github.com/Cay-Zhang/SwiftSpeechExamples) now has a demo called Repeater that simply repeats what you said after each recording, demonstrating the added feature of the release above. You can check that out if you are interested.

**Comment by EnesKaraosman** [comment]
Thanks for your effort 👍
